src/users/src/users-service/repository.py

```diff
@@ -0,0 +1,111 @@
+import random
+from models import User, Address
+import json
+import gzip
+from pynamodb.exceptions import DoesNotExist
+
+claimed_users = {}
+
+def init():
+    try:
+        load_users_into_dynamodb("/src/data/users.json.gz")
+        return True
+    except Exception as e:
+        raise e
+    
+def load_users_into_dynamodb(filename):
+    with gzip.open(filename, 'rt', encoding='utf-8') as file:
+        data = json.load(file)
+        for user_data in data:
+            create_or_update_user(user_data)
+
+def create_or_update_user(user_data):
```

Comment:
Why can't this be an upsert operation using update?

Reply:
true, looks like I was in the process of making the change, commented out and didn't finish it. I'll make the change